mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
yes. It's what I am saying. If you do a bipolar montage on a subset of channels after applying some avg ref proj then you're getting non-avg ref data... and you get this without a warning currently...